Toolkit/ArcGISToolkit/TimeSlider.swift

```diff
@@ -832,20 +845,21 @@ public class TimeSlider: UIControl {
     
     /**
      This will look for all time aware layers which are visible and are participating in time based filtering
-     to initialize slider's fullExtent, currentExtent and timeStepInterval properties.
+     to initialize slider's fullExtent, currentExtent and timeStepInterval properties. Setting observeGeoView
```

Comment:
It seems like we need to indicate in the doc that `observeGeoView` is a property.  Is that accomplished via `#observeGeoView`?

Reply:
Nevermind, forgot we're not in the SDK.